src/libs/ReportUtils.ts

```diff
@@ -1868,8 +1871,18 @@ function getDisplayNameForParticipant(accountID?: number, shouldUseShortForm = f
     }
 
     const personalDetails = getPersonalDetailsForAccountID(accountID);
-    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    const formattedLogin = LocalePhoneNumber.formatPhoneNumber(personalDetails.login || '');
+    if (!personalDetails) {
+        return '';
+    }
```

Comment:
I added it to avoid unnecessary calculations and function calls if `personalDetails` is not available. I think this minor optimization may help in scenarios where the function may be called frequently.